x/nodes/keeper/reward.go

```diff
@@ -7,13 +7,53 @@ import (
 	sdk "github.com/pokt-network/pocket-core/types"
 	govTypes "github.com/pokt-network/pocket-core/x/gov/types"
 	"github.com/pokt-network/pocket-core/x/nodes/types"
+	pcTypes "github.com/pokt-network/pocket-core/x/pocketcore/types"
 )
 
 // RewardForRelays - Award coins to an address using the default multiplier
 func (k Keeper) RewardForRelays(ctx sdk.Ctx, relays sdk.BigInt, address sdk.Address) sdk.BigInt {
 	return k.RewardForRelaysPerChain(ctx, "", relays, address)
 }
 
+// CalculateRelayReward - Calculate the amount of rewards based on the given
+// number of relays and the staked tokens.  The returned reward amount includes
+// DAO & Proposer portions.
+func (k Keeper) CalculateRelayReward(
+	ctx sdk.Ctx,
+	chain string,
+	relays sdk.BigInt,
+	stake sdk.BigInt,
+) (sdk.BigInt, sdk.BigInt) {
+	// feature flags
+	isAfterRSCAL := k.Cdc.IsAfterNamedFeatureActivationHeight(ctx.BlockHeight(), codec.RSCALKey)
+	multiplier := k.GetChainSpecificMultiplier(ctx, chain)
+
+	var coins sdk.BigInt
+	//check if PIP22 is enabled, if so scale the rewards
+	if isAfterRSCAL {
```

Comment:
> 1. NIT for readability:

Instead of having `splitRewards` in two places, I introduced `calculateRewardRewardPip22` to put the PIP-22 logic in.

> 2. I compare this to the previous code and there were no changes, right?

Right, no change in `CalculateRelayReward`.  I don't like horizontal scroll, so I split long lines.